equinix/resource_network_device.go

```diff
@@ -907,6 +918,31 @@ func createVendorConfigurationSchema() map[string]*schema.Schema {
 			Sensitive:   true,
 			Description: neDeviceVendorConfigDescriptions["RootPassword"],
 		},
+		neDeviceVendorConfigSchemaNames["PrivateAddress"]: {
+			Type:        schema.TypeString,
+			Optional:    true,
```

Comment:
Could you add forceNew if change of the attribute will trigger device recreation?